test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -643,6 +659,43 @@ def _assert_artifact_free(output, stray_artifacts):
             artifact, output.stdout
         ), f"Matched {artifact} in {output.stdout} while running {output.command}"
 
+def _test_sm_toolkit_and_ts_version(image, region):
+    """
+    @param image: ECR image URI
+    """
+
+    if "gpu" or "cpu" in image:
+        cmd_smkit = "pip show sagemaker-pytorch-inference | grep -i Version"
+        cmd_ts = "torchserve --version"
+    else:
+        ValueError(
```

Comment:
you need to raise an error here. I would just go with RuntimeError for simplicity - but this should cause the neuron test to fail, but it's currently passing in your CI

Reply:
fixed or no longer an issue